### PR TITLE
add: translation in form component and move interfaces

### PR DESCRIPTION
**Pull request by GustavoUT22**

### CODE REVIEW

File: src/types/ui/Multiselect.ts [hunk 1]
```diff
@@ -10,6 +10,6 @@ export interface IServiceOptions {
 export interface MultiSelectProps {
```
Comment (Mardecera) [inline src/types/ui/Multiselect.ts:10]: The letter "I" is missing, which refers to "Interface".

File: src/lang/en.json [hunk 2]
```diff
@@ -66,18 +66,32 @@
...
+
+  "form.servicios.data.option.grafic.pieces": "Development of graphic pieces",
+  "form.servicios.data.option.branding": "Branding",
+  "form.servicios.data.option.ux.ui.design": "UX/UI Design",
+  "form.servicios.data.option.ux.writing": "UX Writing",
+  "form.servicios.data.option.wep.development": "Web development",
+  "form.servicios.data.option.full.stack.applications": "Full stack applications",
+  "form.servicios.data.option.multi.applications": "Multiplatform applications",
+  "form.servicios.data.option.mobile.applications": "Mobile Development - iOS, Android, Flutter",
+  "form.servicios.data.option.e-commerce": "E-commerce",
```
Comment (Mardecera) [inline src/lang/en.json:89]: e-commerce or ecommerce? To maintain the structure of the key, letters and dots

File: src/components/ContactUs/FormContactUs/FormContactUs.tsx [hunk 3]
```diff
@@ -8,48 +8,13 @@ import { validationSchema } from './FormContactUs.yup'
...
-  { key: '9', name: 'E-commerce' },
-  { key: '10', name: 'Cloud migrations and infrastructure' },
-  { key: '11', name: 'Digital staff' },
-  { key: '12', name: 'SEO Optimization/ SEM' },
-  { key: '13', name: 'Digital Marketing' },
-]
+import { servicesOptions } from './FormData'
+import { IValues } from 'types/ui/Form'
+import { IServiceOptions } from 'types/ui/Multiselect'
+import { MdError } from 'react-icons/md'
```
Comment (Mardecera) [inline src/components/ContactUs/FormContactUs/FormContactUs.tsx:16]: Third party library, this must be above the file, separated by a line.

```
import third party library
/* space */
import our files
```

File: src/types/ui/Multiselect.ts (hunk 1 above)
Comment (GustavoUT22) [reply src/types/ui/Multiselect.ts:10]: Done

File: src/lang/en.json (hunk 2 above)
Comment (GustavoUT22) [reply src/lang/en.json:89]: done

File: src/components/ContactUs/FormContactUs/FormContactUs.tsx (hunk 3 above)
Comment (GustavoUT22) [reply src/components/ContactUs/FormContactUs/FormContactUs.tsx:16]: done

**Review by Mardecera — APPROVED** [review]
Nice!!!!

**Pull request merged**